lib/detectPort.js

```diff
@@ -0,0 +1,17 @@
+'use strict'
+
+const onListen = require('on-net-listen')
+const fs = require('fs')
+
+onListen(function (addr) {
+  this.destroy()
+  const port = Buffer.from(addr.port + '')
+  fs.writeSync(3, port, 0, port.length)
+})
+
+// `nitm` catches the SIGINT so we write it to a file descriptor instead
+const stream = fs.createReadStream('/', { fd: 3 })
+process.on('exit', () => stream.close())
+stream.once('data', (chunk) => {
+  if (chunk.toString() === 'SIGINT') process.exit(0)
+})
```

Comment:
Maybe we should unref the stream here? Just incase the server closes gracefully and we don't wanna keep it open.

Reply:
Good call!

Reply:
Hmm actually not sure how to do this, fs.ReadStream doesn't have an unref(). autocannon will send the SIGINT when it's done anyway so currently the process just keeps running for those additional seconds. Ideally the child process would exit when the server closes and autocannon would also stop, I guess?